.devops/templates/build-test-lint.yml

```diff
@@ -44,7 +44,13 @@ steps:
       DANGER_GITHUB_API_TOKEN: $(DANGER_GITHUB_API_TOKEN)
 
   - script: |
-      yarn buildci $(sinceArg)
```

Comment:
in master this was running agains nothing basically `nx affected --base master --head master`

- it fails CI
- in order to mirror our existing behaviour ( run everything on master on every merge ) we need to use `run-many` => explicit API ✅